src/prachi/Assignment14/ScannerCalculator.java

```diff
@@ -0,0 +1,73 @@
+package prachi.Assignment14;
+
+import java.util.Scanner;
+
+import prachi.Assignment_10.Calculator;
+
+/*Program 2:  Calculator program using Scanner class.*/
+
+public class ScannerCalculator {
+	
+	int add(){
```

Comment:
We should not ask, user to give inputs after calling methods, 
We should ask user to give input while calling methods, and we take input once only from main method not in every method